index.html

```diff
@@ -224,8 +224,12 @@
       </div>
       </div>
       </div>
+<<<<<<< HEAD
       </section>
     
+=======
+    </section>
+>>>>>>> solve bugs
```

Comment:
@Akshay8122 Remove this
![Screenshot 2022-01-17 at 5 09 34 PM](https://user-images.githubusercontent.com/75989919/149763395-1d329214-2801-4b14-9db3-3d784bbc11d6.png)

Reply:
done